data_validation/config_manager.py

```diff
@@ -14,9 +14,12 @@
 
 import copy
 import logging
+from typing import Optional, Union
 
 import google.oauth2.service_account
 from ibis_bigquery.client import BigQueryClient
+import ibis.expr.datatypes as dt
+from third_party.ibis.ibis_oracle.client import OracleClient
```

Comment:
We try/catch the import of OracleClient ([example](https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/develop/data_validation/clients.py#L69)) since we don't require cx_oracle to be installed for non-Oracle validations. 

Instead of importing OracleClient here, you can do `from data_validation import clients` and refer to `clients.OracleClient` in line 717